dev/Repeater/SelectionModel.h

```diff
@@ -116,7 +116,6 @@ class SelectionModel :
 
     // Cached Event args to avoid creation cost every time
     tracker_ref<winrt::SelectionModelChildrenRequestedEventArgs> m_childrenRequestedEventArgs{ this };
```

Comment:
Please verify if m_SelectedIndicesCached will get updated if the collection changes at which point, the indices will need to get updated.

Reply:
Removed this change from SelectionModel

---
In reply to: [347405801](https://github.com/microsoft/microsoft-ui-xaml/pull/1623#discussion_r347405801) [](ancestors = 347405801)